doc/policy_tutorial/DEV_ENV_SETUP.md

```diff
@@ -0,0 +1,94 @@
+# Setting up the development environment
+
+As was clear from the introduction, APIcast policies are created in the Lua programming language. So we need to setup an environment to do some Lua programming. Also, an actual APIcast server would be very nice to perform some local tests.
```

Comment:
```suggestion
You need to configure an environment to work with Lua because APIcast policies are created in the Lua programming language. Additionally, you can use an actual APIcast server to perform local tests.
```